src/Ursa.Themes.Semi/Controls/PathPicker.axaml

```diff
@@ -0,0 +1,84 @@
+﻿<ResourceDictionary xmlns="https://github.com/avaloniaui"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:ursa="https://irihi.tech/ursa">
+    <ControlTheme x:Key="{x:Type ursa:PathPicker}" TargetType="ursa:PathPicker">
+        <Setter Property="Template">
+            <ControlTemplate>
+                <DockPanel HorizontalAlignment="{TemplateBinding HorizontalAlignment}"
+                           VerticalAlignment="{TemplateBinding VerticalAlignment}">
+                    <Button Name="PART_Button"
+                            DockPanel.Dock="Right"
+                            Content="{TemplateBinding Title}"
+                            Margin="1,0,0,0">
+                    </Button>
+                    <TextBox DockPanel.Dock="Left"
+                             Text="{TemplateBinding SelectedPathsText,Mode=TwoWay}">
+                    </TextBox>
+                </DockPanel>
+            </ControlTemplate>
+        </Setter>
+    </ControlTheme>
+
+    <ControlTheme x:Key="PathPickerForMultipleText" TargetType="ursa:PathPicker">
```

Comment:
I think these two themes should be merged, and the "AcceptReturn" should just be controlled by "AllowMultiple"